### ISSUE TITLE
Android build fails because of 'tools:replace="android:allowBackup"'

### ISSUE THREAD

**Issue by otusweb** [iou90/react-native-autoheight-webview#43]
In my AndroidManifest.xml, I specify 
``` android:allowBackup="false"```

Once I added react-native-autoheight-webview, on build I get the following error:
```
Attribute application@allowBackup value=(false) from AndroidManifest.xml:16:7-34
 is also present at [Wazza:react-native-autoheight-webview:unspecified] 
AndroidManifest.xml:12:9-35 value=(true).
Suggestion: add 'tools:replace="android:allowBackup"' to <application> element 
at AndroidManifest.xml:12:5-33:19 to override.
```

I'm new at android development, but it would seem that this plugin should not specify this setting and instead let the app decide. 

following the instruction given in teh build error and adding  ```xmlns:tools="http://schemas.android.com/tools"```
to the manifest fix the issue

**Comment by iou90** [comment]
Should be solved by 58dc1dcbf2556bbca309c0fd7b8be9dada61c663, please try @0.6.0

**Comment by iou90** [comment]
@arcollector removed by 0.6.1
